packages/wonder-blocks-dropdown/components/dropdown-anchor.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+type Props = {|
+    /**
+     * The child element which will act as the opener for the dropdown
+     */
+    children: React.Element<any>,
+
+    /**
+     * Callback used to pass the childs Ref back up to the parent element
+     */
+    anchorRef: (?Element | ?Text | ?HTMLElement) => mixed,
```

Comment:
The arg type of this callback should match `openerElement: ?HTMLElement;` in dropdown.js.  I think `HTMLElement` should be sufficient.  Is there a reason you added `Element` and `Text` here?

Reply:
Yes its because I get a flow error when I pass the anchorNode to the anchorRef prop, flow believes its a Text type.

Reply:
https://github.com/facebook/flow/issues/3548

Reply:
I see.  I think it'd probably be better to make this `openerElement: (?HTMLElement) => mixed` then change
```
const anchorNode = ReactDOM.findDOMNode(this);
```
to be
```
const anchorNode = ((ReactDOM.findDOMNode(this): any): ?HTMLElement);
```
Also, it'd be good to link to that flow ticket directly in the code so people know why we're casting.  Once you cast here the rest of the downstream type-checking should work fine.